site/src/pages/components/popover-hint.research.explainer.mdx

```diff
@@ -0,0 +1,195 @@
+---
+menu: Proposals
+name: Popover=hint and Hover Triggering (Explainer)
```

Comment:
Ok. I went with `Popover Hint and Hover` for now. Not awesome, but at least it's short enough. Likely we should try to remove "(Explainer)" from all of them anyway, since it's quite repetitive and not informative.

@andrico1234 for further comments. But I think ok for this PR.